content/html/concepts/semantic-html/terms/main/main.md

```diff
@@ -0,0 +1,54 @@
+---
+Title: '<main>'
+Description: 'Represents the primary content within the body element of the web page.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Semantic'
+  - 'Tags'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `<main>` semantic tag represents the primary content within the `<body>` of the web page. This excludes content from other structural elements such as the [header](https://www.codecademy.com/resources/docs/html/semantic-html/header) or [footer](https://www.codecademy.com/resources/docs/html/semantic-html/footer).
+
+## Example
+
+Below is an example of the `<main>` tag being used to display a list of notable lines from works by Shakespeare:
+
+```html
+<!DOCTYPE html>
+<html>
+  <head> </head>
+  <body>
+    <header>
+      <img
+        src="https://cdn.pixabay.com/photo/2014/04/03/11/50/drama-312318_640.png"
+        width="200"
+        height="200"
+      />
+    </header>
+    <main>
+      <h1>My favorite Shakespeare lines</h1>
+
```

Comment:
Is this extra space here necessary?

Reply:
I usually like to keep an empty line below my `<hx>` but totally fine with removing it 😄